code/modules/mob/living/carbon/human/species_types/beefmen.dm

```diff
@@ -63,6 +63,17 @@
 	if (inFeatures["beefmouth"] == null || inFeatures["beefmouth"] == "")
 		inFeatures["beefmouth"] = pick(GLOB.mouths_beefman)
 
+/mob/living/carbon/human/proc/adjust_bl_all(var/type = "add", var/amount)
+	for(var/i in bodyparts)
```

Comment:
so, uh, you do realize that this is basically gonna multiply the natural bleeding rate of beefmen by 6, right? because each body part is going to bleed generic_bleedstacks independently?

why not just make their chests bleed?

Reply:
hell, instead of bothering with bleeding rates and such, you could just directly call the bleed() proc

Reply:
I understand this, and because of it, I lowered the amount of bleeding rate for each limb.
What is the bleed() proc by the way?